prepare-release.sh

```diff
@@ -49,11 +49,8 @@ if [ "$PROJECT" == "orm" ] || [ "$PROJECT" == "reactive" ]; then
 	# tags the version
 	# changes the version to the provided development version
 	./gradlew clean releasePrepare -x test --no-scan --no-daemon --no-build-cache \
-		-PreleaseVersion=$RELEASE_VERSION -PdevelopmentVersion=$DEVELOPMENT_VERSION -PgitRemote=origin -PgitBranch=$BRANCH -PdocPublishBranch="production" \
-		-PSONATYPE_OSSRH_USER=$OSSRH_USER -PSONATYPE_OSSRH_PASSWORD=$OSSRH_PASSWORD \
-		-Pgradle.publish.key=$PLUGIN_PORTAL_USERNAME -Pgradle.publish.secret=$PLUGIN_PORTAL_PASSWORD \
-		-PhibernatePublishUsername=$OSSRH_USER -PhibernatePublishPassword=$OSSRH_PASSWORD \
-		-DsigningPassword=$RELEASE_GPG_PASSPHRASE -DsigningKeyFile=$RELEASE_GPG_PRIVATE_KEY_PATH
+		-PreleaseVersion=$RELEASE_VERSION -PdevelopmentVersion=$DEVELOPMENT_VERSION
+		-PgitRemote=origin
```

Comment:
You have removed `-PdocPublishBranch="production"`. Is it because it's not needed at this stage? And also `-PgitBranch=$BRANCH`

Reply:
RIght.  I tried to cleanup what gets passed to just what is needed for that stage.  Happy to add that back if you prefer

Reply:
Well, that's the thing... it is hard to tell.  Between Jenkinsfile -> script -> Gradle its extremely difficult to verify that.  Logically you should not, but it depends on how the script is defined.  But as I looked through the HR Gradle build I believe this is accurate.